org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/CompoundTest.java

```diff
@@ -232,37 +267,37 @@ public void testSimpleCompoundStep3Step() throws Exception {
 		final List<IPosition> points = scan.createPoints();
 
 		// 290K
-		assertEquals(new Double(290), (Double)points.get(0).get("Temperature"));
-		assertEquals(new Double(11),  (Double)points.get(0).get("Y"));
-		assertEquals(new Double(1),   (Double)points.get(0).get("X"));
-		assertEquals(new Double(290), (Double)points.get(1).get("Temperature"));
-		assertEquals(new Double(11),  (Double)points.get(1).get("Y"));
-		assertEquals(new Double(1.6), (Double)points.get(1).get("X"));
-		assertEquals(new Double(290), (Double)points.get(2).get("Temperature"));
-		assertEquals(new Double(11),  (Double)points.get(2).get("Y"));
-		assertEquals(new Double(2.2), (Double)points.get(2).get("X"));
+		assertEquals(new Double(290), points.get(0).get("Temperature"));
+		assertEquals(new Double(11),  points.get(0).get("Y"));
+		assertEquals(new Double(1),   points.get(0).get("X"));
+		assertEquals(new Double(290), points.get(1).get("Temperature"));
+		assertEquals(new Double(11),  points.get(1).get("Y"));
+		assertEquals(new Double(1.6), points.get(1).get("X"));
+		assertEquals(new Double(290), points.get(2).get("Temperature"));
+		assertEquals(new Double(11),  points.get(2).get("Y"));
+		assertEquals(new Double(2.2), points.get(2).get("X"));
 
 		// 291K
-		assertEquals(new Double(291), (Double)points.get(36).get("Temperature"));
-		assertEquals(new Double(11),  (Double)points.get(36).get("Y"));
-		assertEquals(new Double(1),   (Double)points.get(36).get("X"));
-		assertEquals(new Double(291), (Double)points.get(37).get("Temperature"));
-		assertEquals(new Double(11),  (Double)points.get(37).get("Y"));
-		assertEquals(new Double(1.6), (Double)points.get(37).get("X"));
-		assertEquals(new Double(291), (Double)points.get(38).get("Temperature"));
-		assertEquals(new Double(11),  (Double)points.get(38).get("Y"));
-		assertEquals(new Double(2.2), (Double)points.get(38).get("X"));
+		assertEquals(new Double(291), points.get(36).get("Temperature"));
+		assertEquals(new Double(11),  points.get(36).get("Y"));
+		assertEquals(new Double(1),   points.get(36).get("X"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "Double" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2129)